application/controllers/bestitamazoncron.php

```diff
@@ -17,6 +19,23 @@ class bestitAmazonCron extends oxUBase
      */
     protected $_oContainer = null;
 
+    /**
+     * The logger
+     *
+     * @var LoggerInterface
+     */
+    protected $_oLogger;
```

Comment:
That Oxid style ... i know. But I would prefer the normal `$logger`.

Reply:
I don't like it either but this notation signals that the property can be overwritten so its ok in my opinion.

Reply:
A leading `_` point to an internal property / variable. Normally it doesn't say tat you can overwrite the property.